packages/migrate/src/commands/DbPull.ts

```diff
@@ -187,6 +187,12 @@ Set composite types introspection depth to 2 levels
             const providerFromUrl = protocolToConnectorType(`${input.url.split(':')[0]}:`)
             const schema = `${this.urlToDatasource(input.url, providerFromSchema)}\n${removeDatasource(rawSchema)}`
 
+            // Normalize provider from schema for PostgreSQL
+            // For PostgreSQL the protocol can also only be `postgres` which will not match the provider from the schema (which is normalized somewhere already)
+            if(providerFromSchema === 'postgres') {
```

Comment:
```suggestion
            if (providerFromSchema === 'postgres') {
```

Reply:
The above `providerFromSchema` should become a `let` declaration

Reply:
Thanks, the dangers of "coding" in Github UI :D